cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -328,13 +324,24 @@ func (b *bootstrap) ensureMeshConfig() error {
 	return nil
 }
 
-// initiatilizeKubernetesEventsRecorder initializes the generic Kubernetes event recorder and associates it with
+// shouldCreateMeshRootCertificate gets the MeshConfig and returns the values of the enableMeshRootCertificate
+// feature flag
+func (b *bootstrap) shouldCreateMeshRootCertificate() bool {
+	config, err := b.configClient.ConfigV1alpha2().MeshConfigs(b.namespace).Get(context.TODO(), meshConfigName, metav1.GetOptions{})
```

Comment:
can we have this also check if a meshRootCertificate already exists? If there are any we should not create a new one.